README.md

```diff
@@ -23,41 +23,69 @@ The server component to the TourTrak system built using Django.
-2. Setup the postgreSQL database:
+<i>(This is where you begin the server setup instructions)</i>
 
-* switch to the default postgresql user by running `su postgres`
-* create a user w/ read and write permissions: `createuser --pwprompt`
-* Create the DCS database used to collect rider information: `createdb DCS`
+1. Install postgreSQL
+
+	`sudo apt-get install postgresql`
+2. After, install postGIS, which is a postgreSQL extention for handling spatial data
+
+	`sudo apt-get install postgresql-9.3-postgis-2.1`
+3. Setup the postgreSQL database:
+
+* switch to the default postgresql user by running
+
+	`sudo -iu postgres`
+* create a user w/ read and write permissions:
+
+	`createuser --pwprompt`
+* Create the DCS database used to collect rider information:
+
+	`createdb DCS`
 * Setup the postGIS functions:
 
 ```
-  psql -d DCS -f /usr/share/postgresql/9.1/contrib/postgis-1.5/postgis.sql
-  psql -d DCS -f /usr/share/postgresql/9.1/contrib/postgis-1.5/spatial_ref_sys.sql
-  
+  psql DCS
+  CREATE EXTENSION postgis;
+  CREATE EXTENSION postgis_topology;
+  \q
 ```
 
 ## Postgres Database Commands
+<i>(This is not part of the install. Just advice)</i>
 #####Connecting to the database after setup
-1. Type `su postgres`
-2. Type `psql`
-3. Type `\connect DCS`
+1. Type `sudo -iu postgres`
+2. Type `psql DCS`
 
 #####Some helpful commands include:
 1. `\dt` to list all tables
 2. `\q` to quit
-3. `psql -d DCS -f /path/to/script/script.sql` for running scripts without having to connect
+3. `psql DCS < /path/to/script/script.sql` for running scripts without having to connect
+
+
+##Install The Application
 
+1. Install git
 
-##Installation
+	`sudo apt-get install git`
+2. Create a 'devcycle' directory in /usr/local/
+
+	`sudo mkdir /usr/local/devcycle`
+3. Clone this repository into /usr/local/devcycle
+ 
+	`sudo git clone https://github.com/tourtrak/devcycle-server.git /usr/local/devcycle`
+4. <b><font color="red">IMPORTANT:</font> The directory must be named 'devcycle' and not 'devcycle-server'. Django does not support hyphen names for it's applications.</b>
+5. Inside the root directory of your application, install all project dependencies by running our setup script
 
-* Clone this repository into /usr/local/
-* <b><font color="red">IMPORTANT:</font> Rename the directory from devcycle-server to devcycle. Django does not support hyphen names for it's applications.</b>
-* Inside the root directory of your application, install all project dependencies by runnning our setup script `bash setup.sh`
-* Create a virtual host & WSGI file for the Apache server to display the Django application. Open the httpd.conf file 'nano /etc/apache2/httpd.conf'. Copy and paste the following, you may edit 
-these values if desired (such as where to collect static files).
+	`cd /usr/local/devcycle`
+
+	`sudo bash setup.sh`
```

Comment:
@cman131 here's some more for you:

pygobject is a problem. The apt package is python-gobject, but the version may not be a match
smart is also a problem.

Everything else other than what we discussed seems to be fine. The ones we moved over to apt we may need to verify versions and make sure the version apt wants to install is okay.